### PR TITLE
Add function for spliting stereo audio segments into separate mono segments.

### PR DESCRIPTION
**Pull request by dashj**

### CODE REVIEW

File: pydub/audio_segment.py [hunk 1]
```diff
@@ -562,6 +562,18 @@ def set_channels(self, channels):
                                'channels': channels,
                                'frame_width': frame_width})
 
+    def split_to_mono(self):
+        if self.channels == 1:
+            return self
```
Comment (jiaaro) [inline pydub/audio_segment.py]: should return `[self]`

File: pydub/audio_segment.py [hunk 2]
```diff
@@ -562,6 +562,18 @@ def set_channels(self, channels):
...
+
+        left_channel = audioop.tomono(self._data, self.sample_width, 1, 0)
+        right_channel = audioop.tomono(self._data, self.sample_width, 0, 1)
+
+        return [self._spawn(data=left_channel,
```
Comment (jiaaro) [inline pydub/audio_segment.py]: overrides should be: `{'channels': 1, 'frame_width`: self.sample_width}` and in the tests the `frame_count` should not change.

A "frame" contains a sample for each channel (all of which are "played" simultaneously)

**Comment by dashj** [comment]
Ah ok. I've commited the changes.
I was a bit confused about why len() returned half the actual length before the ``frame_width``override.

**Comment by jiaaro** [comment]
Perfect :+1:

**Pull request merged**